packages/document-map/src/document-map.tsx

```diff
@@ -133,6 +144,25 @@ function DocumentMap({
     setRandomId(generateRandomId());
   }, []);
 
+  let args = {
+    canComment: true,
```

Comment:
Is dit niet normaal een instelling die je uit de settings moet halen? Anders is het een beetje gek om eerst `canComment` op true te zetten om vervolgens te gebruiken als default waarde. Of ik begrijp de logica van dit stukje niet helemaal, kan ook

Reply:
Zover ik het begrijp vanuit het comments component is dat ie standaard true is, en dat het overschreven wordt zodra dit niet kan. 
Maar als dit anders / beter moet dan zou ik graag willen weten hoe precies? :)